charts/k8s-job/README.md

```diff
@@ -0,0 +1,359 @@
+# Kubernetes Job Helm Chart
+
+This Helm Chart can be used to deploy your job container under a
+[Job](https://kubernetes.io/docs/concepts/workloads/controllers/job/) resource onto your Kubernetes
+cluster. You can use this Helm Chart to run and deploy a one time job or periodic task such as a security scanner application or data science pipeline job.
+
+
+## How to use this chart?
+
+* See the [root README](/README.adoc) for general instructions on using Gruntwork Helm Charts.
+* See the [examples](/examples) folder for example usage.
+* See the provided [values.yaml](./values.yaml) file for the required and optional configuration values that you can set
+  on this chart.
+
+back to [root README](/README.adoc#core-concepts)
+
+## What resources does this Helm Chart deploy?
+
+The following resources will be deployed with this Helm Chart, depending on which configuration values you use:
+
+- `Job`: A standalone `Job` running the image specified in the
+                `containerImage` input value.
+
+back to [root README](/README.adoc#core-concepts)
+
+## How do I deploy additional resources not managed by the chart?
+
+You can create custom Kubernetes resources, that are not directly managed by the chart, within the `customResources`
+key. You provide each resource manifest directly as a value under `customResources.resources` and set
+`customResources.enabled` to `true`. For examples of custom resources, take a look at the examples in
+[test/fixtures/custom_resources_values.yaml](../../test/fixtures/custom_resources_values.yaml) and
+[test/fixtures/multiple_custom_resources_values.yaml](../../test/fixtures/multiple_custom_resources_values.yaml).
+
+back to [root README](/README.adoc#day-to-day-operations)
+
+## How do I set and share configurations with the application?
```

Comment:
I think we can actually link to the `k8s-service` docs for this section, since the content is largely applicable across the two. Something like:

```
This chart exposes the same parameters as `k8s-service` to set and
share configuration variables with your container Pod. Refer to the
[corresponding section](/charts/k8s-service/README.md#how-do-i-set-and-share-configurations-with-the-application)
in the k8s-service documentation for more details.
```

Reply:
Seconding Yori's comment! Where possible, we like to link to the "single source of truth".